packages/store/addon/-private/system/relationships/state/create.ts

```diff
@@ -24,14 +32,14 @@ function createRelationshipFor(relationshipMeta: RelationshipSchema, store: any,
 }
 
 export default class Relationships {
-  recordData: RelationshipRecordData
-  initializedRelationships: {[key: string]: BelongsToRelationship | ManyRelationship}
-  constructor(recordData) {
-    this.recordData = recordData;
+  _store: Store;
+  initializedRelationships: { [key: string]: BelongsToRelationship | ManyRelationship };
+  constructor(public recordData: RelationshipRecordData) {
     this.initializedRelationships = Object.create(null);
+    this._store = recordData.storeWrapper.store;
```

Comment:
This seems more suspect tbh then us passing a store to our own record data. Maybe we should remove it from the storeWrapper instead? I'm not quite sure it's supposed to be there

Reply:
discussed oob, this is a step towards eliminating in both locations. moving `store` to `_store` to help reduce the leakage from store-wrapper as well until the relationship layer is refactored to not require store.